{"name": "Негативная проверка доверенного номера с символом '%'", "status": "passed", "description": "Проверка неудачного поиска клиента по доверенному номеру с символом '%'", "steps": [{"name": "Проверка сообщения об ошибке и кода ответа", "status": "passed", "start": 1718187369221, "stop": 1718187369221}], "parameters": [{"name": "proofNum", "value": "'996553565311%'"}], "start": 1718187369099, "stop": 1718187369221, "uuid": "52fd184d-c8d2-4ae2-8537-362729b8fb7e", "historyId": "8491f76211981660967405d3cd41fefe", "testCaseId": "a6be3475c37a4f6fbc7e20e79cd9ff1a", "fullName": "207_service_test.TestProofNumSuite#test_get_info_client_has_special_symbol_in_proofNum_get_error_response", "labels": [{"name": "suite", "value": "(207 сервис) Получение информации по клиенту"}, {"name": "subSuite", "value": "Проверки по доверенному номеру"}, {"name": "host", "value": "PC-411-140322D"}, {"name": "thread", "value": "9896-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "207_service_test"}]}